src/pages/EditMonitor.vue

```diff
@@ -422,7 +422,7 @@
                                 <div class="form-text">{{ $t('snmpOIDHelptext') }} </div>
                             </div>
 
-                            <div v-if="monitor.type === 'snmp'" class="my-3">
+                            <div v-if="monitor.type === 'snmp' && monitor.snmpVersion !== '3'" class="my-3"">
```

Comment:
why?

Reply:
This condition avoids mixing SNMPv1/v2c and SNMPv3 configuration.
v1/v2c are community-string based, while SNMPv3 uses a user-based security model, so v3-specific fields (like username) are only shown when snmpVersion === '3'.
This prevents invalid configurations and keeps the UI clear.

Reply:
that does not explain why you want to guard the selector for `snmpVersion` to be only visible when `snmpVersion !== '3'`.

So I select v3, see that it is not the right version and would back to v2.. not possible.

I think you meant

```suggestion
                            <div v-if="monitor.type === 'snmp'" class="my-3">
```

Reply:
You’re right — good catch.
The SNMP version selector should remain visible so users can switch back from v3.
I’ve applied the suggested change.